packages/shared_preferences/shared_preferences_linux/example/pubspec.yaml

```diff
@@ -5,14 +5,12 @@ dependencies:
   flutter:
     sdk: flutter
   shared_preferences: any
```

Comment:
Is this actually used, or can we remove it? It may be cruft from a previous iteration. Generally we don't want the platform implementations depending on the app-facing platform.

Reply:
It is needed as the integration test depends on it :

https://github.com/flutter/plugins/blob/aa827e30ba37af3f2c49645b337dbe67c3ba9dbe/packages/shared_preferences/shared_preferences_linux/example/integration_test/shared_preferences_test.dart#L3

Reply:
That should be fixed; see https://github.com/flutter/plugins/blob/master/packages/shared_preferences/shared_preferences_windows/example/integration_test/shared_preferences_test.dart for changes. This package predates the realization that having implementation package tests depend on the app-facing package is a problem.